src/components/Demo/DemoCreate/DemoCreateForm/DemoCreateFormItem/index.tsx

```diff
@@ -103,6 +115,9 @@ export const DemoCreateFormItem: FunctionComponent<DemoCreateFormItemProps> = ({
           classNameContainer="py-5 border-t border-cloud-300"
           heading={formItem.title}
           headingTag="h3"
+          isOpen={openIndex === formItemIndex}
```

Comment:
```suggestion
          openIndex={openIndex === formItemIndex}
```

Reply:
if change to number the component wont know it should open or not